CONTRIBUTING.md

```diff
@@ -167,13 +167,13 @@ any PMC member:
 1. Merge the release proposal PR into the master branch of the repository. There
    should not be any conflicts as the text in the files that have changed should
    only be changed during this release process.
-1. Create a new branch called **"v$major.$minor"** from the **Release SHA**.
 1. Cherry pick the commit in which the release proposal PR was merged, to pick
```

Comment:
Do we still need to cherry pick it since we're still on master it is has the release PR already?

Reply:
Wait I think I still need a line like this - to avoid merging any changes that went onto master after the RC was created. Update en route

Reply:
Since we're just using tags now, shouldn't we just create a PR with all release changes and merge it into master? I know other things "could" get merged but I wonder if it would just be easier to just freeze master while we're in the release process. Otherwise, don't you need two commits for the version string updates (one for the tag and one for master)?

Reply:
There is still stuff to do after the release PR is merged into master though. Pushing an updated version now. LMK what you think